buildSrc/src/main/groovy/org/elasticsearch/gradle/test/CopyRestApiSpecPlugin.groovy

```diff
@@ -0,0 +1,40 @@
+package org.elasticsearch.gradle.test
+
+import org.elasticsearch.gradle.VersionProperties
+import org.elasticsearch.gradle.info.BuildParams
+import org.elasticsearch.gradle.testclusters.RestTestRunnerTask
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.Task
+import org.gradle.api.tasks.Copy
+
+/**
+ * Copies the core REST specs to the local resource directory for the REST YAML tests.
+ */
+class CopyRestApiSpecPlugin implements Plugin<Project> {
```

Comment:
We still have a few places where we need the specs copied to the local resources. Notably the examples so that they can rely on the release jar. There are few others projects where it is easier to copy them locally then run from the global dir. However, I believe all (but 1) of the modules, plugins, and x-pack plugins run from the global dir.

Reply:
It's not clear to me what the difference is between just putting them on the test runtime classpath, and copying them to the resources output directory. Either way, we are just throwing these files on the classpath.

Reply:
I think I may have mis-attributed "easier", I will re-visit the usage of this. However, this is still required for the examples project.